README.md

```diff
@@ -1,2 +1,155 @@
-# bedrock-web-data-hub
-Web-based Bedrock Data Hub
+# Web Based Bedrock Data Hub Adapter _(bedrock-web-data-hub)_
```

Comment:
I think "Client" would be better than "Adapter"

Reply:
Done.